### PR TITLE
adding more detail to staking and delegating

### PR DESCRIPTION
**Pull request by joshuahannan**
## Description

Reorganized some info about registration and adding new transactions to get node and delegator info

### CODE REVIEW

**Comment by joshuahannan** [comment]
@psiemens or @robmyers can you review this?

File: docs/content/token/staking/locked/delegators.mdx [hunk 1]
```diff
@@ -24,9 +24,13 @@ transaction with the following arguments:
 | **id**     | `String` | The ID of the node to delegate to. |
 | **amount** | `UFix64` | The number of FLOW tokens to delegate. |
 
-This transaction attaches a `NodeDelegatorProxy` capability to the `TokenHolder` resource. 
+This transaction registers the account as a delegator to the node ID they specified
+and attaches a `NodeDelegatorProxy` capability to the `TokenHolder` resource. 
 This capability can later be used to perform delegation actions.
 
+Once an account has registered as a delegator, they cannot register a second time.
```
Comment (robmyers) [inline docs/content/token/staking/locked/delegators.mdx:31]:
```suggestion
Once an account has registered as a delegator with one node, they cannot register with any others. For more information, see "Register to stake for a different node" below.
```

File: docs/content/token/staking/locked/delegators.mdx [hunk 2]
```diff
@@ -112,3 +117,15 @@ transaction with the following arguments:
...
+
+## Getting Info about an accounts delegation
+
+To return a struct with all the information associated with a delegator, anyone
+can use the **Get Delegator Info** ([TH.19](../transactions/#token-holder)) script with the following arguments:
+
+| Argument   | Type     | Description |
+|------------|----------|-------------|
+| **nodeID** | `String` | The 32 byte ID of the node of the delegator |
+| **nodeID** | `UInt8` | The ID of the delegator |
```
Comment (robmyers) [inline docs/content/token/staking/locked/delegators.mdx:129]:
```suggestion
| **nodeID** | `UInt8`  | The ID of the delegator |
```

**Review by robmyers — APPROVED** [review]
Looking good! I suggested mangling one sentence and adding a space.

**Pull request merged**